python/rmm/rmm/allocators/torch.py

```diff
@@ -29,7 +29,7 @@
         # is pure Python and will therefore be in the source directory.
         # Instead, we search relative to an arbitrary file in the compiled
         # package. We use the _lib.lib module because it is small.
-        from rmm._lib import lib
+        from rmm.cpp import lib
```

Comment:
Is this import *only* to get the path? Is there a cleaner way to do that?   lib.pyx/pxd are nearly empty, so I could see them disappearing.